src/modules/ocaml.rs

```diff
@@ -29,6 +30,18 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 _ => None,
             })
             .map(|variable| match variable {
+                "switch" => {
+                    let stdout = context.exec_cmd("opam", &["switch", "-s", "show"])?.stdout;
+                    let switch_raw = stdout.trim();
```

Comment:
Just invoke trim on stdout directly. There is no need to introduce an additional variable.

Reply:
i did this originally, but:

```
temporary value dropped while borrowed
consider using a `let` binding to create a longer lived value (rustc E0716)
```

the extra var is consistent w/ the other impls as well